### PR TITLE
caom2-artifact-sync tool

### PR DESCRIPTION
**Pull request by brianmajor**

### CODE REVIEW

File: caom2-artifact-sync/src/main/java/ca/nrc/cadc/caom2/artifactsync/Main.java [hunk 1]
```diff
@@ -143,11 +151,32 @@ else if (am.isSet("cert"))
...
 
+            int batchSize = ArtifactHarvester.DEFAULT_BATCH_SIZE;
+            if (am.isSet("batchsize"))
+            {
+                try
+                {
+                    batchSize = Integer.parseInt(am.getValue("batchsize"));
+                    if (batchSize < 1 || batchSize > 10000)
+                    {
+                        log.error("value for --threads must be between 1 and 10000");
```
Comment (javierduranarenas) [inline caom2-artifact-sync/src/main/java/ca/nrc/cadc/caom2/artifactsync/Main.java]: Typo. Should be --batchSize

File: caom2-artifact-sync/src/main/java/ca/nrc/cadc/caom2/artifactsync/Main.java [hunk 2]
```diff
@@ -240,11 +300,13 @@ private static void usage()
     {
         StringBuilder sb = new StringBuilder();
         sb.append("\n\nusage: caom2-artifact-sync [-v|--verbose|-d|--debug] [-h|--help] ...");
-        sb.append("\n     --artifactStore= fully qualified class name");
-        sb.append("\n     --database= <server.database.schema>");
-        sb.append("\n     --threads= number  of threads to be used to import artifacts (default: 1)");
+        sb.append("\n     --artifactStore=fully qualified class name");
+        sb.append("\n     --database=<server.database.schema>");
```
Comment (javierduranarenas) [inline caom2-artifact-sync/src/main/java/ca/nrc/cadc/caom2/artifactsync/Main.java]: Should we evaluate the definition of --database format? dot-separated-values could be problematic for some databases.

Comment (javierduranarenas) [inline caom2-artifact-sync/src/main/java/ca/nrc/cadc/caom2/artifactsync/Main.java]: Should we evaluate the definition of --database format? dot-separated-values could be problematic for some databases.

File: caom2-artifact-sync/src/main/java/ca/nrc/cadc/caom2/artifactsync/Main.java (hunk 1 above)
Comment (javierduranarenas) [inline caom2-artifact-sync/src/main/java/ca/nrc/cadc/caom2/artifactsync/Main.java]: Typo. Should be --batchSize

Comment (brianmajor) [reply caom2-artifact-sync/src/main/java/ca/nrc/cadc/caom2/artifactsync/Main.java]: Thanks, good catch.

File: caom2-artifact-sync/src/main/java/ca/nrc/cadc/caom2/artifactsync/Main.java (hunk 2 above)
Comment (brianmajor) [reply caom2-artifact-sync/src/main/java/ca/nrc/cadc/caom2/artifactsync/Main.java]: Yes, but the caom2persistence library knows how to handle this format.

**Pull request merged**